raspberry_pi_installation.md

```diff
@@ -280,18 +286,26 @@ Open the terminal window and enter the following commands...
 
     $ curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
     $ sudo apt-get install nodejs
-
+    $ sudo npm i -g npm
```

Comment:
Node.js installation installs also npm and it is already installed - this command would fail if it were not. So why add this?

Reply:
Yes You are right that "Node.js installation installs also npm" but what I have **experienced** is that when running 
$ curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
$ sudo apt-get install nodejs
for **update** purposes npm is not upgraded ?

Reply:
These instructions are for installing Node.js, not updating it.

Reply:
Sorry, don't understand ?? The text is below the heading **Update Node and NPM**
https://github.com/MatsA/signalk-server-node/blob/master/raspberry_pi_installation.md

Reply:
Oh sorry. The document has `sudo apt-get install nodejs` in two places, I thought this one was the first where you install Node.js.

Sigh. This is why I don't like detailed _how to update_ guides, because there rarely is just one recipe to fit all cases. Afaik these instructions are good for upgrading from a Node.js version <8, but if you have 8 and just want to upgrade to the latest version (including npm) you should `apt-get upgrade nodejs`. Can you somehow confirm this?

Reply:
No problem. No can't confirm, but we can change to "apt-get upgrade nodejs" and with that You will stay on 8.x ? 
If all updates was via the Admin GUI, it had solved it ;-)

Reply:
Yes, let's change to that.